light-rest-4j/src/main/java/com/networknt/codegen/rest/SwaggerGenerator.java

```diff
@@ -31,6 +31,14 @@
     //static ObjectMapper mapper = new ObjectMapper();
 
     private Map<String, String> typeMapping = new HashMap<>();
+
+    private static List<String> javaKeyWordList = new ArrayList<>(Arrays.asList("abstract", "continue", "for",	"new",
+            "switch", "assert", "default", "goto", "package", "synchronized", "boolean", "do", "if", "private", "this",
```

Comment:
Seems like you have different implementations of detecting java keywords between openapi and swagger now

Reply:
@NicholasAzar I forgot to update the swagger one, I will modify the swagger one to sync with openapi